packages/@lwc/template-compiler/src/__tests__/fixtures/attributes/attribute-crossorigin/expected.js

```diff
@@ -5,9 +5,9 @@ const $fragment3 = parseFragment`<audio src="http://www.example.com/video.mp3" c
 function tmpl($api, $cmp, $slotset, $ctx) {
   const { st: api_static_fragment } = $api;
   return [
-    api_static_fragment($fragment1(), 1),
-    api_static_fragment($fragment2(), 3),
-    api_static_fragment($fragment3(), 5),
+    api_static_fragment($fragment1(), 0),
+    api_static_fragment($fragment2(), 1),
+    api_static_fragment($fragment3(), 2),
```

Comment:
The main affect from moving the `databag` is that the keys change, they line up with what I would expect from looking at the html now.